app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::SheltersController < ApplicationController
+  def index
+    @shelters = Shelter.reverse_abc
```

Comment:
beautiful encapsulation of business logic into model methods! These controllers are so nice and clean